hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java

```diff
@@ -596,6 +596,14 @@ protected void updateCostsWithAction(Cluster cluster, Action action) {
     }
   }
 
+  private void addCostFunction(CostFunction costFunction) {
+    if (costFunction.getMultiplier() <= 0) {
+      LOG.info("exclude cost function {}", costFunction.getClass().getSimpleName());
```

Comment:
This only gets logged once when the config is first loaded or updated.The function loaded is already reported at https://github.com/apache/hbase/blob/9e017e261ed323c25e9c36c36f8ee628e855830a/hbase-server/src/main/java/org/apache/hadoop/hbase/master/balancer/StochasticLoadBalancer.java#L224

I can change this into debug mode since the summary is already reported.

Reply:
I believe it's better to have this logic:
```
    if (costFunction.getMultiplier() > 0) {
      LOG.debug("Including cost function {}", costFunction.getClass().getSimpleName());
      costFunctions.add(costFunction);
    }
```

Reply:
@clarax Oh, right. Since there's already a summary log at info that lists out all the loaded functions, I'd say omit the debug logging entirely. You good with that plan @virajjasani ?

Reply:
Yeah, sounds better.

Reply:
Seems changes are incorporated here https://github.com/apache/hbase/pull/2386

@ndimiduk @huaxiangsun